implementations/typescript/ockam/ockam_app/src/components/share/CreateServiceInvitation.svelte

```diff
@@ -0,0 +1,116 @@
+<script>
+  import { invoke } from '@tauri-apps/api/tauri';
+
+  let project_id;
+  let recipient_email;
+
+  let project_identity;
+  let project_route;
+  let project_authority_identity;
+  let project_authority_route;
+  let shared_node_identity;
+  let shared_node_route;
+
+  let projects = invoke('plugin:sharing|list_projects');
```

Comment:
Where does this command structure come from? It's autogenerated that way when you load commands from a plugin?

Reply:
> It's autogenerated that way when you load commands from a plugin?
Yes, that's exactly right, and I assume was there to provide namespacing in case multiple plugins wanted to offer similarly named commands without collisions.

https://next--tauri.netlify.app/next/guides/features/plugin

Reply:
That's interesting. I had to give very explicit names to tauri commands and this solves that problem. I'll open a PR refactoring the other components into plugins (@etorreborre let me know if that sounds good to you).